### ISSUE TITLE
Beginner-friendly issues.

### ISSUE THREAD

**Issue by suryapratapsinghsuryavanshi** [suryapratapsinghsuryavanshi/NASA-Media-Search#1]
This is a simple issue for ReactJs or NodeJs developers to make a contribution to it.

**Comment by Khusyasy** [comment]
hello @suryapratapsinghsuryavanshi can i get more info on this issue?

**Comment by suryapratapsinghsuryavanshi** [comment]
Yes, we have added this issue for all those who are just a beginner in open source contribution, this issue is mainly focused on beginners, and the application is based on React+Redux, so more users can contribute to it. can. @Khusyasy

**Comment by jaspreet11111111** [comment]
hello @suryapratapsinghsuryavanshi so what is the issue, means what should we do for contributing to it?

**Comment by suryapratapsinghsuryavanshi** [comment]
Hii @jaspreet11111111, Make UI improvements and also improve the code quality for test cases.
